spec/controllers/projects/results_controller_spec.rb

```diff
@@ -120,12 +128,18 @@
         end
       end
     end
+
+    describe 'without project no access' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.